### ISSUE TITLE
Installing Datadog.Trace.1.1.0 via nuget fails to install LibLog 5.0.6

### ISSUE THREAD

**Issue by jesanfafon** [DataDog/dd-trace-dotnet#328]
I have a .NET Framework 4.5.2 project that I'm upgrading from 0.7.1-beta to 1.1.0.

My installation is failing with the following:
```
Attempting to gather dependency information for multiple packages with respect to project 'SkySlope.DataFeed.Api', targeting '.NETFramework,Version=v4.5.2'
Gathering dependency information took 10.85 sec
Attempting to resolve dependencies for multiple packages.
Resolving dependency information took 0 ms
Resolving actions install multiple packages
Retrieving package 'Datadog.Trace 1.1.0' from 'nuget.org'.
Retrieving package 'LibLog 5.0.6' from 'nuget.org'.
Retrieving package 'MsgPack.Cli 1.0.1' from 'nuget.org'.
  GET https://api.nuget.org/v3-flatcontainer/datadog.trace/1.1.0/datadog.trace.1.1.0.nupkg
Removed package 'Datadog.Trace 0.7.1-beta' from 'packages.config'
  GET https://api.nuget.org/v3-flatcontainer/msgpack.cli/1.0.1/msgpack.cli.1.0.1.nupkg
Successfully uninstalled 'Datadog.Trace 0.7.1-beta' from SkySlope.DataFeed.Api
  OK https://api.nuget.org/v3-flatcontainer/msgpack.cli/1.0.1/msgpack.cli.1.0.1.nupkg 12ms
Installing MsgPack.Cli 1.0.1.
Removed package 'MsgPack.Cli 1.0.0' from 'packages.config'
Successfully uninstalled 'MsgPack.Cli 1.0.0' from SkySlope.DataFeed.Api
Install failed. Rolling back...
Package 'LibLog 5.0.6' does not exist in project 'SkySlope.DataFeed.Api'
Package 'MsgPack.Cli 1.0.0' already exists in folder 'C:\skyslope\skyslope-datafeed\packages'
Added package 'MsgPack.Cli 1.0.0' to 'packages.config'
Package 'Datadog.Trace 0.7.1-beta' already exists in folder 'C:\skyslope\skyslope-datafeed\packages'
Added package 'Datadog.Trace 0.7.1-beta' to 'packages.config'
Package 'LibLog 5.0.6' does not exist in folder 'C:\skyslope\skyslope-datafeed\packages'
Executing nuget actions took 527.85 ms
Update-Package : Could not install package 'LibLog 5.0.6'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.5.2', but the package does not contain any 
assembly references or content files that are compatible with that framework. For more information, contact the package author.
At line:1 char:1
+ Update-Package Datadog.Trace
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Update-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.UpdatePackageCommand
 
Time Elapsed: 00:00:11.4693147
```
Any ideas on how to proceed? We'll fall back to 1.0.0 for now, since that doesn't have LibLog as a dependency.

**Comment by colin-higgins** [comment]
Hi @jesanfafon 
I just gave it a shot locally and was able to pull down the v1.1.0 nuget into a .NET 4.5.2 class library.
Based on the LibLog source, it looks like LibLog supports net45 (https://github.com/damianh/LibLog/blob/master/src/LibLog/LibLog.csproj)

![image](https://user-images.githubusercontent.com/1801443/56441532-6be3ac00-62bb-11e9-97ea-c483334807c0.png)
![image](https://user-images.githubusercontent.com/1801443/56441537-700fc980-62bb-11e9-967f-3169bd093b0e.png)
![image](https://user-images.githubusercontent.com/1801443/56441542-743be700-62bb-11e9-8a5a-957225eeb293.png)

If there is any more information you think might be relevant, please include it here. 
I'll look into this more.

**Comment by jesanfafon** [comment]
I tried to recreate your test and got the same error as in my original post. I also installed VS2017 community edition in case this was an error only affecting VS2019, but I'm getting the same issue. 

I'll see if I can get more verbose logs from nuget and maybe better triage this problem.

**Comment by jesanfafon** [comment]
Oops, I accidentally closed this

**Comment by jesanfafon** [comment]
Bisecting the issue, it looks like LibLog lost support for framework 4.5.2 in version 5.0.5 after restoring it in 5.0.2

Adding it back in 5.0.2: https://github.com/damianh/LibLog/issues/167
My issue on their repo: https://github.com/damianh/LibLog/issues/237

For the interim, what do you think about using LibLog 5.0.4 as a workaround for Framework 4.5.2 support?

**Comment by jesanfafon** [comment]
As per the author of LibLog, they only support the new format csproj files, which only support upgrading a narrow range of project types. None of my current projects using DataDog are convertible to the new project format. 

Currently, I'm unable to use the latest version of DataDog for any of my projects.

**Comment by lucaspimentel** [comment]
This is a bug in the way we added this NuGet package. The `LibLog` package does not contain compiled assemblies like most packages. It contains source code that is compiled into our project and should not be a downstream dependency in your applications. We will resolve this in the next bugfix release.

**Comment by jesanfafon** [comment]
Awesome! Should I close this then?

**Comment by lucaspimentel** [comment]
We will close it when the issue is resolved the and fix is released. Thanks for reporting this, btw.

**Comment by lucaspimentel** [comment]
This issue should be resolved with [release 1.2](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.2.0).
